submissions/Oleksandr-Heleta/OOP_exercise/index.js

```diff
@@ -31,32 +41,38 @@ class Animal extends Inhabitants {
 }
 
 class Dog extends Animal {
-   constructor(name, gender, say) {
-      super('dog', name, gender, say);
+   constructor(name, gender) {
+      super('dog', name, gender, "bark");
    }
 }
 
 class Cat extends Animal {
-   constructor(name, gender, say) {
-      super('cat', name, gender, say);
+   constructor(name, gender) {
+      super('cat', name, gender, "meow");
    }
 }
 
-const dog = new Dog('Bob', "male", "bark");
-const cat = new Cat('Kitty', "female", "meow");
+class CatWoman extends Human {
+   constructor(name, gender) {
+      super(name, gender)
+      this.mutation = new Cat();
```

Comment:
This way you create a cat within a cat-woman. Doesn't reflect how it works in real world.

The following from the previous review is still valid.
> Static class properties/methods are to the rescue.

Please work on comments in their entirety and ask specific questions if anything is not clear.